docs/products/valkey.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Aiven for Valkey
+---
+
+Aiven for Valkey is a managed, in-memory NoSQL database service that offers high performance, scalability, and security.
+
+Developed under the Linux Foundation, Valkey is an open-source fork of Redis® designed
+to provide a seamless and reliable alternative to Redis OSS.  Aiven for Valkey
+ensures full compatibility with Redis OSS 7.2.4 and Aiven for Caching, making it easy
+for users to transition their existing applications without disruption.
+
+:::tip
+Since Aiven for Valkey is the new open-source fork of Redis®, and Aiven for Caching
+is essentially the same NoSQL service with a new name, users can continue to
+refer to the Aiven for Caching documentation for detailed instructions on
+using the service.
+:::
+
+## Key Features
+
+- **Open source**: Valkey is licensed under the permissive BSD-3 license,
+  ensuring open-source availability.
+- **Redis compatible**: Fully compatible with Redis OSS 7.2.4 and
+  Aiven for Caching, providing a seamless transition for users.
+- **High performance**: As an in-memory NoSQL database, Valkey offers fast data
+  retrieval with low latency, ideal for applications requiring real-time data processing.
+- **Managed service**: Aiven handles maintenance, updates, and monitoring,
+  freeing you from database management complexities.
+- **Migration**: Support for a migration path from external Redis® implementations and
```

Comment:
we wont have Aiven Caching -> Aiven Valkey by our GA date unfortunately. only 3rd party

Reply:
I thought this was already in place - https://aiven.atlassian.net/browse/MA-2713